Images_And_Graphics_style_v1.css

```diff
@@ -333,7 +372,12 @@ ul li:hover ul {
 }
 
 .logo1 {
-	float: right;
+	float: left;
+}
+
+.Tawa_College_Logo {
+	margin-left: 300px;	
+	margin-top: -10px;

```

Comment:
I added a new class for my tawa college logo so I can add it at the bottom of my website just above the footer